src/python_testing/matter_testing_support.py

```diff
@@ -707,6 +708,16 @@ def check_pics(self, pics_key: str) -> bool:
         pics_key = pics_key.strip().upper()
         return pics_key in picsd and picsd[pics_key]
 
+    def openCommissioningWindow(self, dev_ctrl: ChipDeviceCtrl, node_id: int, discriminator) -> CommissioningParameters:
+        try:
+            params = dev_ctrl.OpenCommissioningWindow(nodeid=node_id, timeout=900, iteration=1000,
+                                                      discriminator=discriminator, option=1)
```

Comment:
Could you please remove the discriminator from the input parameters and just use random.randint(0,4095). That can be passed back as a tuple. This will stop people from using the original discriminator (which is what I did originally). That value isn't populated unless you commission the device, which the TH doesn't do (see https://github.com/project-chip/connectedhomeip/pull/31046 for an explanation).

Reply:
Got it! I did it in that way so now it returns as tuple of (CommissioningParameters and random discriminator)